packages/astro/src/@types/astro.ts

```diff
@@ -1375,6 +1375,91 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		devOverlay?: boolean;
+
+		// TODO review with docs team before merging to `main`
+		/**
+		 * @docs
+		 * @name experimental.i18n
+		 * @type {object}
+		 * @version 3.5.0
+		 * @type {object}
+		 * @description
+		 *
+		 * Configures experimental i18n routing and allows you to specify some customization options.
+		 */
+		i18n?: {
+			/**
+			 * @docs
+			 * @name experimental.i18n.defaultLocale
+			 * @type {string}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * The default locale of your website/application
+			 */
+			defaultLocale: string;
+			/**
+			 * @docs
+			 * @name experimental.i18n.locales
+			 * @type {string[]}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * A list of all locales supported by the website. e.g. `['en', 'fr', 'es']` This list should/should not include the `defaultLocale`.
+			 */
+			locales: string[];
+
+			/**
+			 * @docs
+			 * @name experimental.i18n.fallback
+			 * @type {Record<string, string>}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * The content fallback strategy when navigating to pages in non-default languages that do not exist. (e.g. a translated page has not been created).
+			 *
+			 * This a plain object, where a key is a locale that hasn't traslated pages, and the corresponding value is the locale where you want to redirect the missing page to.
```

Comment:
```suggestion
			 * The fallback strategy when navigating to pages that do not exist (e.g. a translated page has not been created).
			 *
			 * Use this object to declare a fallback `locale` route for each language you support. If no fallback is specified, then unavailable pages will return a 404.
```
Just notice this suggestion didn't get merged in the other PR, and I'm suggesting again another tiny tweak to it:

I don't think introducing the idea of "content" here is actually needed! Now that I've seen how this works, and especially given we're talking about which pages to navigate to or to 404, I don't think this is a "content" strategy. This allows us to save that word in case you do end up doing something like the docs that *does* inject some fallback content on a page (at the requested route).

I also think we can simplify the wording by eliminating "non-default languages" and just say, what to do when a page doesn't exist. Which is a nicer sentence for everyone!

So what do you think about the above?